### ISSUE TITLE
Node-webcl soon superseeded with node_opencl

### ISSUE THREAD

**Issue by Bondifrench** [mil-tokyo/sushi#29]
@mikeseven is indicating that node-webcl will be soon superseeded by https://github.com/mikeseven/node-opencl.
If we replace node-webcl with node-opencl, can we make Sushi benefit from it?
Thx

**Comment by mikeseven** [comment]
Yes absolutely. The programming model of node-opencl is just opencl, so it supports latest OpenCL specs, which WebCL couldn't. Adapting your library should be rather straigtforward.
